src/lib.rs

```diff
@@ -677,17 +732,22 @@ impl PagerState {
                     ),
                     len_line_number,
                     idx,
+                    #[cfg(feature = "search")]
+                    &mut search_idx,
                 )
             })
             .collect::<Vec<String>>();
 
+        #[cfg(feature = "search")]
+        {
+            self.search_idx.append(&mut search_idx);
```

Comment:
Wouldn't you just want to do `self.search_idx = search_idx`? If you change the search term, wouldn't this cause the matches from both of the searches to be contained in the search_idx?

Reply:
Oh! I guess I did it by mistake. I will fix it in the morning.

Reply:
[9404e55](https://github.com/arijit79/minus/pull/62/commits/9404e55974b7aaf44d0adc5c55987763a72d8937) resolves this.